test/integration/bulk-load-test.js

```diff
@@ -521,6 +521,72 @@ describe('Bulk Load Tests', function() {
       done();
     }
   });
+  it('should throw `RequestError: Canceled` after 10ms', function(done) {
+
+    const bulkLoad = connection.newBulkLoad('#tmpTestTable5', { keepNulls: true }, function(err, rowCount) {
+      if (err) {
+        assert.strictEqual(err.name, 'RequestError');
+        assert.strictEqual(err.message, 'Canceled.');
+        return done();
+      }
+
+      done(new Error('Test did not throw error as expected'));
```

Comment:
I think you can simplify these assertions here like this:

```suggestion
      assert.instanceOf(err, Error);
      assert.strictEqual(err.name, 'RequestError');
      assert.strictEqual(err.message, 'Canceled.');

      return done();
```

Can you also fix this similarly in the other tests you added? 🙇‍♂️